doc/user/content/ingest-data/striim.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Striiim"
+description: "How to connect Striiim to Materialize as a source."
+aliases:
+  - /integrations/striiim/
+menu:
+  main:
+    parent: "kafka"
+    name: "Striiim"
+---
+
+This guide walks through the steps to ingest data from Striim into Materialize using a [Kafka Source](/sql/create-source/kafka/).
+
+Striim is a real-time data integration platform that offers a variety of connectors for databases, messaging systems, and other data sources. This guide focuses on streaming data from MySQL to Kafka but can be adapted to other databases as well.
+
+## Before you begin
+
+Ensure you have the following:
+
+- An active [Striim account](https://go2.striim.com/free-trial) and operational service.
+- Configured MySQL, Oracle, or SQL Server with streamable tables.
+- An operational Kafka cluster.
+- Confluent Schema Registry set up for Avro schema management.
+
+## Integration Steps
+
+### Database Source Configuration
+
+1. **MySQL CDC Setup:**
+    - Activate binary logging with `row` format in MySQL.
+    - Follow Striim's [MySQL CDC guide](https://www.striim.com/docs/GCP/StriimForBigQuery/en/prerequisite-checks-mysql.html) for detailed setup instructions.
+
+2. **Striim MySQLReader App Creation:**
+    - Deploy an app within Striim, entering MySQL specifics and selecting target tables for streaming.
+
+Alternatively, follow the [Oracle CDC guide](https://www.striim.com/docs/en/oracle-database-cdc.html) or [SQL Server CDC guide](https://www.striim.com/docs/en/sql-server-cdc.html) for database-specific integration steps.
+
+### Kafka Target Configuration
+
+1. **Kafka Cluster Verification:**
+    - Create a [Kafka Writer](https://www.striim.com/docs/en/kafka-writer.html) within Striim to direct data to the Kafka cluster.
+    - For the 'Input Stream' select the data source created in the previous step.
+    - Specify the Kafka broker details and topic name.
+    - Under the 'Advanced settings', in the 'Kafka Config' field, add the `value.serializer=io.confluent.kafka.serializers.KafkaAvroSerializer` configuration to encode records in the Avro format.
```

Comment:
```suggestion
    - Under the 'Advanced settings', in the 'Kafka Config' field, add the `value.serializer=io.confluent.kafka.serializers.KafkaAvroSerializer` configuration to encode records using the Confluent wire protocol.
```

(They'll come out as Avro either way; the salient bit is using `io.confluent...` to get the confluent wire protocol.)